arraycontext/impl/pyopencl.py

```diff
@@ -338,6 +268,9 @@ def from_numpy(self, array: np.ndarray):
         return cla.to_device(self.queue, array, allocator=self.allocator)
 
     def to_numpy(self, array):
+        from numpy import isscalar
+        if isscalar(array):
```

Comment:
Why this instead of `isinstance(array, numbers.Number)`? (Change or add a comment.)

Reply:
No particular reason, changed.